appeals/web/src/server/appeals/appeal-details/service-user/__tests__/__snapshots__/service-user.test.js.snap

```diff
@@ -0,0 +1,388 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`service-user GET /change/:userType should fail to render the change page when the service user type is not a valid string 1`] = `
```

Comment:
The 'should' is a little vague here - it might be better to be more specific about what should be rendered when it fails (in this case it looks like the 500 error page)